openstack/service.yaml

```
@@ -831,15 +841,18 @@ topology_template:
         ports:  ['9093:9093']
         exposed_ports:  ['9093']
         docker_network_name:  { get_property: [ SELF, network, name ] }
+        volumes:
+          - "/etc/alertmananger:/etc/alertmanager"
       requirements:
         - host: docker-host
         - registry: docker-public-registry
         - network: docker-network
+        - configuration: alertmanager-config
```

```suggestion
        - dependency: alertmanager-config
```
Docker container module does not have `configuration `requirement